### DISCUSSION TITLE
highlight heading title

### DISCUSSION THREAD

**Discussion by dchinmay2** [nvim-neorg/neorg#287] [Q&A] [answered]
How do i highlight heading titles?
setting it to
```lua
heading = {
  ["1"] = "+mygroup"
}
```
highlights only the contents under the heading like this
![image](https://user-images.githubusercontent.com/36493671/147723302-63385e53-5cb2-471b-9ae8-9a03b13e0d93.png)
and setting it to
```lua
heading = {
  ["1"] = { title = "+mygroup"}
}
```
has no effect (I thought of doing this after grepping for TSAttribute)

**Comment by cccra** [comment, marked as answer]
I was frustrated by this for a while. For me what seemed to fix it was capitalising "heading" and "title", like this

```lua
Heading = {
	['1'] = { Title = '+mygroup' }
}
```

**Reply by dchinmay2** [reply]
Thanks
